ld/armv7m7-imxrt106x.ldt

```diff
@@ -28,17 +28,21 @@
 /* FlexRAM configuration */
 #if defined(CUSTOM_FLEXRAM_CONFIG)
 FLEXRAM_CONFIG = CUSTOM_FLEXRAM_CONFIG;
+FLEXRAM_ITCM_BANKS = CUSTOM_FLEXRAM_ITCM_BANKS;
+FLEXRAM_DTCM_BANKS = CUSTOM_FLEXRAM_DTCM_BANKS;
 #else
 FLEXRAM_CONFIG = 0xaaaaabff;
+FLEXRAM_ITCM_BANKS = 5;
+FLEXRAM_DTCM_BANKS = 11;
 #endif
 
 /* Memory map setup */
 MEMORY
 {
 	/* TODO: use FLEXRAM_CONFIG value to setup ocram/itcm/dtcm partitioning (*32k) */
-	m_itcm    (rwx) : ORIGIN = 0x00000000, LENGTH = 5 * 32k
+	m_itcm    (rwx) : ORIGIN = 0x00000000, LENGTH = FLEXRAM_ITCM_BANKS * 32k
 	m_itext   (rwx) : ORIGIN = 0x00026800, LENGTH = 6 * 1k
```

Comment:
btw. `m_itext` as the subregion of `m_itcm` is at the end of `m_itcm` region, so the config is misleading.